### ISSUE TITLE
input_datetime bug in Lovelace

### ISSUE THREAD

**Issue by Anders-McLean** [home-assistant/frontend#13582]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The input_datetime card in lovelace is completely bugged out. If you have several date inputs on your dashboard you can only set ONE date, then they all freeze up and don't accept more inputs until you reload the page. Except for the "today" button witch still works. It also doesn't work to manually type the date. I can do it by closing the input picker but it wont accept the date after i reload the page. So if I want to choose a date 1 year into the future I have to click the next month button 12 times... Come on. 

### Describe the behavior you expected

I expect to be able to change the input date as many times as I want without it breaking and also to be able to manually enter the date. 

### Steps to reproduce the issue

Just add several input_datetime entities to a dashboard. 

### What version of Home Assistant Core has the issue?

2022.8.7

### In which browser are you experiencing the issue with?

Chrome and Edge, newest 

### Which operating system are you using to run this browser?

Windows 11

**Comment by orange-artworks** [comment]
I have noticed similar behaviour (the date picker of input_datetime not working consistently).  By my observation the picker can't be used to choose a date if you load the picker by clicking on the calendar icon.  If you click on the date itself the picker works.  This has been the same on Edge, Chrome (Windows) and iOS through all versions since I've used it (1 year).  Like the description above the "today" button is the only one that works when you've loaded the date picker by clicking the calendar icon.
